docs/install-kafka-operator.md

```diff
@@ -38,18 +38,15 @@ This method uses a command-line tool of the commercial [Banzai Cloud Supertubes]
 
 {{< kafka-operator >}} uses [cert-manager](https://cert-manager.io) for issuing certificates to clients and brokers. Deploy and configure cert-manager if you haven't already done so.
 
-> Note:
->
-> - {{< kafka-operator >}} 0.8.x and newer supports cert-manager 1.3.x
```

Comment:
I'd keep this so those who use koperator version pre 0.18.1 know that they need to use cert-manager 1.3.x version